frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionDataSource/QuestionDataSource.styled.tsx

```diff
@@ -9,3 +10,13 @@ export const TablesDivider = styled.span`
   padding: 0 0.2em;
   user-select: none;
 `;
+
+export const TableInfoIcon = styled(_TableInfoIcon)`
+  color: ${color("text-light")};
+  visibility: visible;
+  font-size: min(1rem, 1em);
+  padding: 0;
+  margin-left: 0.5em;
```

Comment:
Shouldn't we use `rem` here?

Reply:
The same thing applies here as for [the icon size](bcd283cd2f77a6ae8a75b0616bbf7ef2fa24c6e9) we want the margin to be smaller in contexts where the icon gets rendered small.er